test/controllers/password_resets_controller_test.rb

```diff
@@ -31,7 +31,10 @@ class PasswordResetsControllerTest < ActionDispatch::IntegrationTest
 
     email = @person.email
 
-      # set the user agent in the request headers
+    # Test can flap on CI if we don't clear the queue
+    clear_enqueued_jobs
+
+    # set the user agent in the request headers
```

Comment:
thank you!! I saw this flappy test a couple times in dev but wasn't able to figure it out yet